tamr_unify_client/models/attribute_configuration/collection.py

```diff
@@ -0,0 +1,75 @@
+from tamr_unify_client.models.attribute_configuration.resource import (
+    AttributeConfiguration,
+)
+from tamr_unify_client.models.base_collection import BaseCollection
+
+
+class AttributeConfigurationCollection(BaseCollection):
+    """Collection of :class'~tamr_unify_client.models.attribute_configuration.resource.AttributeConfiguration~'
```

Comment:
Needs backticks instead of quotes (:class\`~tamr_unify_client.models.attribute_configuration.resource.AttributeConfiguration\`)